EIPS/eip-introspection.md

```diff
@@ -0,0 +1,59 @@
+---
+eip: TBD
```

Comment:
```suggestion
eip: 7814
```
Assigning next sequential EIP/ERC/RIP number.
Numbers are assigned by editors & associates.

Please also update the filename.